eng/globalconfigs/Common.globalconfig

```diff
@@ -0,0 +1,121 @@
+is_global = true
+
+# Default severity for all IDE code style rules with category 'Style'
+dotnet_analyzer_diagnostic.category-Style.severity = warning
+
+# Default severity for all IDE code quality rules with category 'CodeQuality'
+dotnet_analyzer_diagnostic.category-CodeQuality.severity = warning
+
+### Configuration for RSxxxx rules ###
+
+# RS1024: Compare symbols correctly
+# https://github.com/dotnet/roslyn-analyzers/issues/3389
```

Comment:
Do we need a new tracking issue to re-enable the rule on the repo once we move to a newer package which has the fix for #3389?

Reply:
```suggestion
# https://github.com/dotnet/roslyn-analyzers/issues/6381
```